wordpress-mobile/WordPress-Android
# Feature/use view binding in posts

This PR introduces view binding in the posts package. This includes the History fragment and prepublishing nudges. Overall it's the same as all the other changes

To test:
- Go to an existing post
- Go to history
- Publish a post
- Check the prepublishing bottom sheet works as expected

PR submission checklist:
